### PR TITLE
Add activity monitor bundle 🔩️

### PR DESCRIPTION
**Pull request by daniel-rose**
- add method get all activities
- add method get one activity by id
- add methods to handle activity status (start, restart, stop, ...)
- map pm2 format to bundle specific format
- generate classes by transfer.json
- ...

### CODE REVIEW

File: packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+        return $pm2Activities->getArrayCopy();
+    }
+
+    /**
+     * @param int $activityId
+     *
+     * @return \Jellyfish\ActivityMonitor\Pm2Interface
+     */
+    public function startActivity(int $activityId): Pm2Interface
```
Comment (julianzimmermann) [inline packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php:74]: why int in and cast to string?

File: packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+        return $this;
+    }
+
+    /**
+     * @param int $activityId
+     *
+     * @return \Jellyfish\ActivityMonitor\Pm2Interface
+     */
+    public function restartActivity(int $activityId): Pm2Interface
```
Comment (julianzimmermann) [inline packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php:86]: same

File: packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+        return $this;
+    }
+
+    /**
+     * @param int $activityId
+     *
+     * @return \Jellyfish\ActivityMonitor\Pm2Interface
+     */
+    public function stopActivity(int $activityId): Pm2Interface
```
Comment (julianzimmermann) [inline packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php:98]: same

**Review by julianzimmermann — APPROVED** [review]

File: packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php (hunk 1 above)
Comment (daniel-rose) [reply packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php:74]: The parameter "command" of the process method "create" allows only an array of string.

**Pull request merged**